src/core/utils/helpers/currency.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Formats a number as a Danish Krone (DKK) currency string using the "da-DK" locale.
+ *
+ * @param {number} number - The numeric value to format into a currency string.
+ * @returns {string} A string formatted in Danish Krone currency format.
+ */
```

Comment:
I am not against using jsdoc, but since we are not using it anywhere else, it sticks out I bit I think. I could also indicate that we have tooling that uses it - which we have not 😃

Reply:
@spaceo I have changed it 👍🏻